discovery-provider/src/app.py

```diff
@@ -332,13 +332,17 @@ def configure_celery(flask_app, celery, test_config=None):
     celery.conf.update(
         imports=["src.tasks.index", "src.tasks.index_blacklist",
                  "src.tasks.index_plays", "src.tasks.index_metrics",
-                 "src.tasks.index_materialized_views",
+                 "src.tasks.index_materialized_views", "src.tasks.vacuum_db",
                  "src.tasks.index_network_peers", "src.tasks.index_trending",
                  "src.tasks.cache_user_balance", "src.monitors.monitoring_queue",
                  "src.tasks.cache_trending_playlists", "src.tasks.index_solana_plays",
                  "src.tasks.index_aggregate_views", "src.tasks.index_challenges"
                  ],
         beat_schedule={
+            "vacuum_db": {
```

Comment:
nit - we try to preserve order with the imports and beat schedule, this should go below update_materialized_views